app/bundles/LeadBundle/EventListener/FormSubscriber.php

```diff
@@ -32,11 +37,34 @@ class FormSubscriber implements EventSubscriberInterface
     private $emailModel;
 
     /**
+<<<<<<< HEAD
```

Comment:
@hluchas how did you miss this? @dongilbert can you resolve this? Also, removing several classes is not considered a BC break?

Reply:
Probably because it was in comments - I know that's how I missed it. 

I'll document the removed classes specifically.